modules/core/src/test/java/org/apache/ignite/internal/processors/cache/binary/GridCacheBinaryObjectsAbstractSelfTest.java

```diff
@@ -423,30 +425,96 @@ public void testMap() throws Exception {
     }
 
     /**
+     * Checks deserialization of elements in the singleton map.
+     *
+     * @throws Exception If failed.
```

Comment:
Exception is never thrown. It is shown by IDEA.

Reply:
Fixed